cmd/livepeer/livepeer.go

```diff
@@ -202,14 +202,17 @@ func updateNilsForUnsetFlags(cfg starter.LivepeerConfig) starter.LivepeerConfig
 	return res
 }
 
-// extendTimeouts extends transcoding timeouts for the testing purpose.
+// streamTesterMode extends transcoding timeouts and disable caching for the testing purpose.
 // This functionality is intended for Stream Tester to avoid timing out while measuring orchestrator performance.
-func extendTimeouts() {
-	if boolVal, _ := strconv.ParseBool(os.Getenv("LP_EXTEND_TIMEOUTS")); boolVal {
+func streamTesterMode() {
+	if boolVal, _ := strconv.ParseBool(os.Getenv("LP_STREAM_TESTER_MODE")); boolVal {
```

Comment:
Hmm, from this env name I imagined it would be like a string which could have several modes. WDYT of renaming this to `LP_IS_STREAM_TESTER` or something like that?

Reply:
Actually, to be even more specific, WDYT of `LP_IS_ORCH_TESTER` instead, since stream-tester is something more general than that?

Reply:
Good idea, renamed to `LP_IS_ORCH_TESTER` in d9835b6490e37c7f14ebaf36c60d73af92696568